ports/libmariadb/portfile.cmake

```
@@ -21,6 +21,11 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     iconv WITH_ICONV
 )
 
+if("-DWITH_SSL=ON" IN_LIST FEATURE_OPTIONS)
+    list(REMOVE_ITEM FEATURE_OPTIONS "-DWITH_SSL=ON")	
+    list(APPEND FEATURE_OPTIONS "-DWITH_SSL=OPENSSL")
+endif()

```

```suggestion
if("openssl" IN_LIST FEATURES)
    set(WITH_SSL=OPENSSL)
else()
    set(WITH_SSL=OFF)
endif()
```
Remove line 20 and add `-DWITH_SSL=${WITH_SSL}` to line 33.